src/library/basetracktablemodel.cpp

```diff
@@ -1041,3 +1041,17 @@ TrackId BaseTrackTableModel::doGetTrackId(
         const TrackPointer& pTrack) const {
     return pTrack ? pTrack->getId() : TrackId();
 }
+
+bool BaseTrackTableModel::updateTrackGenreText(
```

Comment:
This code is duplicated multiple times, but I guess we can't add it to `trackmodel.h` because it don't have a reference to the track collection manager?

Reply:
`TrackCollectionManager` stores the custom tag mapping config and is only available in implementations.